tools/vsimporter/src/PBX/PBXNativeTarget.cpp

```diff
@@ -73,13 +80,25 @@ String PBXNativeTarget::getProductFileType() const {
         return "";
 }
 
+String PBXNativeTarget::getEntitlementsPath() const {
+    // Path should be in both debug and release, so just go with debug
+    const XCBuildConfiguration* config = m_nativeBuildConfigurationListPtr->getConfiguration("Debug");
+    const VariableCollection& buildSettings = config->getBuildSettings();
+
+    // If the value exists, ret will be set, o/w it will stay empty string
+    String ret = "";
+    buildSettings.getValue("CODE_SIGN_ENTITLEMENTS", ret);
+    return ret;
+}
```

Comment:
SBResourcesBuildPhase.cpp does this simpler. See SBResourcesBuildPhase::writeVCProjectFiles:
`String plistPath = bs.second->getValue("INFOPLIST_FILE");`